src/contract/ContractCreateFlow.js

```diff
@@ -254,4 +258,80 @@ export default class ContractCreateFlow {
 
         return response;
     }
+
+    /**
+     * @param {Signer} signer
+     * @returns {Promise<TransactionResponse>}
+     */
+    async executeWithSigner(signer) {
+        if (this._bytecode == null) {
+            throw new Error("cannot create contract with no bytecode");
+        }
+
+        if (signer.getAccountKey == null) {
+            throw new Error(
+                "`Signer.getAccountKey()` is not implemented, but is requried for `ContractCreateFlow`"
```

Comment:
Small typo in "required" (hope this isn't too 'nit-picky' :) )

Reply:
Heh, good catch, and I'm completely fine with nit-picking.